docs/SRS/SRS.tex

```diff
@@ -640,72 +640,72 @@ \subsection{Functional Requirements}
-\textbf{Fit Criterion:} Users should be able to successfully enter their email and password without encountering errors during the registration process.\\
+\textbf{Description:} The system must provide a clear and intuitive interface for account creation, where users can easily enter their email and password.\\
+\textbf{Rationale:} A registration interface is crucial for reducing the entry barrier for new users, enhancing user satisfaction, and decreasing the likelihood of registration abandonment.\\
+\textbf{Fit Criterion:} The interface should allow users to enter their email and password and complete the registration in no more than three steps, with each step clearly delineated.\\
 
 \medskip
 \noindent \textbf{Requirement ID:} FR2\\
 \textbf{Use Case:} 2\\
-\textbf{Description:} The system must validate user credentials (email and password) when a user attempts to sign in.\\
-\textbf{Rationale:} This requirement is necessary to fulfill the authentication process for registered users.\\
-\textbf{Fit Criterion:} Users with valid credentials should be granted access to the application, while users with invalid credentials should receive an error message.\\
+\textbf{Description:} The system must securely validate user credentials like email and password during the sign-in process.\\
+\textbf{Rationale:}  Secure and reliable authentication is essential for protecting user accounts and sensitive data from unauthorized access and complying with data protection regulations.\\
+\textbf{Fit Criterion:} Users with valid credentials must be granted access to the application, while users with invalid credentials must be denied access and receive a clear, non-ambiguous error message regarding the specific authentication failure within 3 seconds of their attempt.\\
 
 \medskip
 \noindent \textbf{Requirement ID:} FR3\\
 \textbf{Use Case:} 6, 7, 8, 9\\
-\textbf{Description:} The system must support the functionality to capture and process receipt data from user-submitted photos.\\
-\textbf{Rationale:} This requirement allows users to add new spending data.\\
-\textbf{Fit Criterion:} Users should be able to submit a photo of a receipt, and the system should successfully process the data from the receipt and store it in the database.\\
+\textbf{Description:} The system must support the capture and digitization of receipt data from user-uploaded photos. This involves the ability to accurately recognize, extract, and process text and numerical data from various receipt formats and layouts.\\
+\textbf{Rationale:} Enabling users to upload photos of receipts and automatically converting these into digital spending data significantly enhances user convenience by reducing manual data entry.\\
+\textbf{Fit Criterion:} The system must successfully recognize and extract data from uploaded receipt photos with an accuracy rate of at least 90\% under typical lighting and photo quality conditions.\\
```

Comment:
I understand why we're setting it to 90\%  but can we say "over 90\%"? Just because accuracy with a financial app is probably the most important factor.

Reply:
Do we having some requirement for "allowing user to fix things when in the 10% failure scenario"?

Reply:
idk. btw resolved